code/game/turfs/simulated/dirtystation.dm

```diff
@@ -36,25 +39,36 @@
 	name = "dusty robot debris"
 	desc = "Looks like nobody has touched this in a while."
 
-
 //Making the station dirty, one tile at a time. Called by master controller's setup_objects
 
 /turf/open/floor/proc/MakeDirty()
 	if(prob(66))	//fastest possible exit 2/3 of the time
 		return
 
-	// These look weird if you make them dirty
+	//What is object-oriented programming? istype() chains are baaad unless it's an attackby. Now turf has a "dirtiness" variable that determines this kind of thing.
```

Comment:
This is pointing out why the legacy code below it shouldn't be replicated. Should I remove that legacy code?